VKSample/VKSample/Modules/FriendPhotosGalleryModule/PhotosGalleryCollectionViewController.swift

```diff
@@ -60,20 +61,30 @@ final class PhotosGalleryCollectionViewController: UICollectionViewController {
         networkService.fetchUsersPhoto(ownerID: friendID) { [weak self] result in
             guard let self = self else { return }
             switch result {
-            case let .success(photoString):
-                let photos = photoString
-                var photosNames: [String] = []
-                for item in photos {
-                    photosNames.append(item.url)
-                }
-                self.photoURLNames = photosNames
+            case let .success(photo):
+                self.photos = photo
                 self.collectionView.reloadData()
             case let .failure(error):
                 self.showErrorAlert(title: Constants.errorTitleString, message: "\(error.localizedDescription)")
             }
         }
     }
 
+    private func loadPhotosFromRealm() {
+        do {
+            guard let objects = realmService.readData(items: Photo.self) else { return }
+            let userId = objects.map(\.ownerID)
+            if userId.contains(where: { id in
```

Comment:
попробуй второй guard сделать
```suggestion
            if userId.contains(where: { id in
```

Reply:
не очень понимаю как я могу тут это условие в guard положить и completion выполнить